dts/bindings/rtc/nxp,irtc.yaml

```diff
@@ -0,0 +1,33 @@
+# Copyright 2024 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+description: IRTC
+
+compatible: "nxp,irtc"
+
+include: rtc.yaml
+
+properties:
+  reg:
+    required: true
+
+  clock-select:
+    type: int
+    required: true
+    description: |
+      The input clock select for IRTC.
+      0 <- 16.384 kHz
+      1 <- 32.768 kHz
+
+  alarms-count:
+    type: int
+    required: true
+    description: Number of alarms supported by IRTC.
+
+  output-clk-en:
+    type: boolean
+    description: |
+      The IRTC can send its clock as an output source
+      for other peripherals to use.
```

Comment:
what other peripherals?

Reply:
Any other peripheral that can use the RTC freq as an Input Clock Source.